src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,38 @@
+package lotto.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+    private final List<Integer> numbers;
```

Comment:
```suggestion
    private final Set<LotoNumber> numbers;
```
- List 대신에 Set을 사용해주면 어떨까요?
- Integer 대신에 LottoNumber 라는 원시값 포장 객체를 만들어서 사용해주면 어떨까요?